atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractLockAwareTransactionManager.java

```diff
@@ -124,7 +124,9 @@
     }
 
     private void logSuccess(UUID runId, int failureCount) {
-        log.info("[{}] Successfully completed transaction after {} retries.", runId, failureCount);
+        if (failureCount > 0) {
+            log.info("[{}] Successfully completed transaction after {} retries.", runId, failureCount);
```

Comment:
as a general style thing, I was very surprised to find atlas inheriting loggers from superclass - it makes this logline look like it's coming from somewhere else in the code